src/key.rs

```diff
@@ -1070,7 +1078,10 @@ impl CPtr for KeyPair {
     fn as_mut_c_ptr(&mut self) -> *mut Self::Target { &mut self.0 }
 }
 
-/// An x-only public key, used for verification of schnorr signatures and serialized according to BIP-340.
+/// An x-only public key - the X co-ordinate of a point on the secp256k1 curve.
+///
+/// To convert to a point (`PublicKey`) one must choose the [`Parity`] of the Y co-ordinate. When
+/// used to create schnorr signatures the Y co-ordinate is always positive.
```

Comment:
I think `coordinate` shouldn't have a `-`. I would also say "Taproot signature" rather than "schnorr signature".

Reply:
re: schnorr vs taproot I thought here in secp we used schnorr and in bitcoin used "taproot". There are zero other occurrences of "taproot" currently in this crate (except some secp-sys test code)? I'm not arguing for or against just asking.